### ISSUE TITLE
Initial values from app.properties when using jdbcStateRepository

### ISSUE THREAD

**Issue by amb-sebastian-podgorski-pt** [togglz/togglz#1184]
I have JDBCStateRepository and EnumBasedFeatureProvider. I added my feature switch to applicatiom.yaml with initial value set to true but I can see it is not added to db at app initializing and the value on console is false as the default value is taken based on presence of annotation @EnabledByDefault. 

Is it possible to set initial value within app properties with jdbc state repo? When I am using InMemoryStateRepo then it is working fine

**Comment by ewoelfel** [comment]
@amb-sebastian-podgorski-pt  you are absolutely right. Thank you for the hint. I created https://github.com/togglz/togglz/pull/1376 that reproduces the issue.
